Bottom: 543f20c60682e6847151071b999d1a0b438cdb4e
Top:    57443424e966f87819d9fcaf8b4d3735be50661d
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-03 08:45:03 -0500

No need to delete /etc/mailname before overwrite


---
diff --git a/docker/docker-exim4/docker-entrypoint.sh b/docker/docker-exim4/docker-entrypoint.sh
index e6fb458d..8d9d35cb 100755
--- a/docker/docker-exim4/docker-entrypoint.sh
+++ b/docker/docker-exim4/docker-entrypoint.sh
@@ -25,8 +25,6 @@ case ${1} in
           && [[ ! -f ${CONFIG_PATH}/exim4.conf ]]
         then
             # set the hostname
-            [ -e /etc/mailname ] \
-                && rm -rf /etc/mailname
             echo "${HOSTNAME}" > /etc/mailname
             # copy certificate
             [ -e /etc/ssl/private/exim.crt ] \
